### ISSUE TITLE
⌨ VoiceOver not announcing verification

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#16048]
### 🛡️comment from fable tester: 
When I linked the skill to the relevant experience and added it, Voiceover didn't announce that it had been done, it just repeated the name of the button to connect the skill to the experience. I had to navigate down the page to make sure the experience had been added. 

### Expected behaviour: 
There should be some sort of verification of the experience being connected to the skill. 
_It's possible_ that this issue is that the toast isn't being read by VoiceOver. 

### 📋 Steps to Reproduce

1. sign in 
2. [apply to this recruitment process](https://uat-talentcloud.tbs-sct.gc.ca/en/jobs/a81b87db-388b-4ac5-9c3b-f3c47bbee7ca)
3. complete the steps to get to the skills requirements 
4. connect a career timeline experience to the required skills with missing experience association. 

### What operating systems are you encountering the problem on?

macOS

### What browsers are you encountering the problem on?

Safari

### What assistive technologies are you encountering the problem on?

VoiceOver

### ✅ Acceptance Criteria

- [ ] VoiceOver announces that the experience has been linked or the toast is announced.
